ft-email-auth-provider/src/handlers/create_account.rs

```diff
@@ -119,10 +132,12 @@ pub fn create_account(
     mut conn: ft_sdk::Connection,
     ft_sdk::Form(payload): ft_sdk::Form<CreateAccountPayload>,
     ft_sdk::Cookie(sid): ft_sdk::Cookie<{ ft_sdk::auth::SESSION_KEY }>,
+    // code can be invaild. eg: xyz
```

Comment:
This is the most important function so keep it at the top.

Reply:
done in https://github.com/fastn-community/auth/pull/7/commits/305104323d6e67c183d70fa1abec3253c48944b8